src/Migrator.EF6.Tools/CommandBase.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if NET451
+﻿#if NET462
```

Comment:
I can retraget the package to NET46 if you like. But like you wrote, it has to be 4.6.X.